pkgs/by-name/po/pom/package.nix

```
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "pom";
-  version = "0-unstable-2024-04-29";
+  version = "0.1.0";
```

```suggestion
  version = "0.1.0-unstable-2024-05-17";
```

https://github.com/maaslalani/pom/releases/tag/v0.1.0 refers to a8a2da7043f222b9c849d1ea93726433980469c0.

---

Thanks!